plugins/register-component/src/components/RegisterComponentPage/RegisterComponentPage.test.tsx

```diff
@@ -15,20 +15,46 @@
  */
 
 import React from 'react';
-import { render } from '@testing-library/react';
-import mockFetch from 'jest-fetch-mock';
+import { render, cleanup } from '@testing-library/react';
 import RegisterComponentPage from './RegisterComponentPage';
 import { ThemeProvider } from '@material-ui/core';
 import { lightTheme } from '@backstage/theme';
+import { errorApiRef, ApiProvider, ApiRegistry } from '@backstage/core';
+import { catalogApiRef } from '@backstage/plugin-catalog';
+import { MemoryRouter } from 'react-router-dom';
 
+const errorApi = { post: () => {} };
+const catalogApi: jest.Mocked<typeof catalogApiRef.T> = {
+  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+  addLocation: jest.fn((_a, _b) => new Promise(() => {})),
```

Comment:
Could also just not specify a and b at all, `()`

Reply:
This will block forever though right

Reply:
Then the type is failing since it's expecting 2 args for a function :)